nservicebus/azure-service-bus/index.md

```diff
@@ -11,37 +11,37 @@ related:
 
-[Azure Service Bus (ASB)](https://azure.microsoft.com/en-us/services/service-bus/) is a messaging service (broker) that sits between applications, allowing them to exchange messages in a loosely coupled fashion for improved scale and resiliency. Service Bus Queues offer simple first in, first out guaranteed message delivery and support a range of standard protocols (REST, AMQP, WS*) and APIs (to put/pull messages on/off the queue). Service Bus Topics deliver messages to multiple subscribers and easily use fan-out pattern to deliver messages to downstream systems.
+Note: Publish/Subscribe and Timeouts (including message deferral) are supported natively by the ASB transport and do not require NServiceBus persistence.
 
- * The main advantage of ASB is that it offers a highly reliable and (relatively) low latency remote messaging infrastructure. A single message can be up to 256 KB in size, and a queue can keep many messages at once, up to 5 GB size in total. Furthermore it is capable of emulating local transactions using its queue peek-lock mechanism. NServiceBus is an abstraction over ASB. It takes advantage of ASB's built-in features, such as message deduplication and deferred messages, and provides higher-level, convenient API for programmers on top of ASB.
- * The main disadvantage of ASB is its dependency on TCP (for low latency), which may require opening some outbound ports on the firewall. Additionally, in some systems the price may get high ($1 per million messages).
+ * The main advantage of ASB is that it offers a highly reliable and low latency remote messaging infrastructure. A single message can be up to 256 KB in size, and a queue can store many messages at once, up to 5 GB size in total. Furthermore, it is capable of emulating local transactions using its queue [peek-lock mechanism](https://msdn.microsoft.com/en-us/library/azure/hh780722.aspx). 
+ * The main disadvantage of ASB is its dependency on TCP (for low latency), which may require opening outbound ports on the firewall. Additionally, in some systems the price for the service may get high ($1 per million messages).
 
-Note: Publish/Subscribe and Timeouts (including message deferral) are supported natively by the ASB transport and do not use persistence.
+NOTE: As part of the Azure support for NServiceBus, one can choose between two transports provided by the Azure platform: Azure Storage Queues and Azure Service Bus. Each of these two options has separate features, capabilities, and usage characteristics. A detailed comparison and discussion of when to select which is beyond the scope of this document. To help decide which option best suits the application's needs, review the Azure article [Azure Queues and Azure Service Bus Queues - Compared and Contrasted](https://azure.microsoft.com/en-us/documentation/articles/service-bus-azure-and-service-bus-queues-compared-contrasted/).
```

Comment:
That MSFT reference is good, I made some changes. [PR](https://github.com/Azure/azure-content/pull/6512)